packages/newspringchurchapp/src/prayer/index.js

```diff
@@ -11,7 +11,7 @@ class PrayerMenuConnected extends PureComponent {
 
   render() {
     return (
-      <Query query={getPrayerMenuCategories}>
+      <Query query={GET_PRAYER_MENU_CATEGORIES}>
```

Comment:
Do we plan on updating every query in CORE as well? Because CORE doesn't use this style.

Reply:
I could ask, I think we should. if they were going to not merge in our stuff based on best practices, it makes sense to not stop halfway.

Reply:
I wonder if we should wait on CORE to begin this specific query name style change...